ephios/api/urls.py

```diff
@@ -15,4 +24,58 @@
         get_schema_view(title="ephios", description="ephios API", version="1.0.0"),
         name="openapi-schema",
     ),
+    path(
+        "settings/",
+        include(
+            [
+                # Token management views
+                path(
+                    "token/",
+                    AccessTokensListView.as_view(),
+                    name="settings-access-token-list",
+                ),
+                path(
+                    "token/create/",
+                    AccessTokenCreateView.as_view(),
+                    name="settings-access-token-create",
+                ),
+                path(
+                    "token/reveal/<int:pk>/",
+                    AccessTokenRevealView.as_view(),
+                    name="settings-access-token-reveal",
+                ),
+                path(
+                    "token/revoke/",
+                    AccessTokenRevokeView.as_view(),
+                    name="settings-access-token-revoke",
+                ),
+                # Application management views
+                path(
+                    "applications/",
+                    staff_required(AllUserApplicationList.as_view()),
+                    name="settings-oauth-app-list",
+                ),
+                path(
+                    "applications/register/",
+                    staff_required(views.ApplicationRegistration.as_view()),
+                    name="settings-oauth-app-register",
+                ),
+                path(
+                    "applications/<int:pk>/",
+                    staff_required(views.ApplicationDetail.as_view()),
+                    name="settings-oauth-app-detail",
+                ),
+                path(
+                    "applications/<int:pk>/delete/",
+                    staff_required(views.ApplicationDelete.as_view()),
```

Comment:
when deleting an application, the success url is wrong and results in a NoReverseMatch at /api/settings/applications/1/delete/

Reverse for 'list' not found. 'list' is not a valid view function or pattern name.

Reply:
damnit I'll just write test for these too

Reply:
fixed